data/Plug.css

```diff
@@ -1,4 +1,9 @@
-button.auto-timezone:checked {
+switch.auto-timezone:checked {
+    background-color: @GRAPE_500;
+    border: 1px solid transparent;
```

Comment:
Please don’t make these kinds of style changes

Reply:
It's not a matter of style change with evil intent, just bad targetting with CSS, fixing ASAP.